test/CompoundStaking.test.ts

```diff
@@ -1,14 +1,16 @@
 import { waffle, ethers } from "hardhat"
 import { expect } from "chai"
 import { BigNumber, BigNumberish, constants, Contract, Wallet } from 'ethers'
-const { AddressZero } = constants
-import { stakingFixture } from "./utilities/fixtures"
-import { mineBlocks, expandTo18Decimals } from "./utilities/index"
+import { compoundFixture } from "./utilities/fixtures"
+import { mineBlocks, setTimestamp, expandTo18Decimals, time } from "./utilities/index"
 
 import { ERC20 } from "../types/ERC20"
 import { Staking } from "../types/Staking"
+import exp from "constants"
 
-describe("Staking", () => {
+
+
+describe("Compound", () => {
```

Comment:
Why did we rename everything back to Compound? Let's stick with the Staking naming, this way we don't commit to one particular implementation and still describe the entity properly - the main feature is staking, not compounding